wire-ios/Wire-iOS/Sources/Helpers/SemanticColors.swift

```diff
@@ -191,6 +191,27 @@ public enum SemanticColors {
         static let backgroundPickUp = UIColor(light: Asset.Colors.green500Light, dark: Asset.Colors.green500Dark)
         static let textUnderlineEnabledDefault = UIColor(light: Asset.Colors.black, dark: Asset.Colors.white)
     }
+
+    public enum SketchColors {
```

Comment:
For localizable strings, you use "drawing" and here "sketch". Maybe it's worse to use the same names. What do you think?

Reply:
Done!